src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -238,7 +237,7 @@ public Builder workspace(FilePath workspace) {
      * @return A reference to the {@link Builder}.
      */
     public Builder namespace(String namespace) {
-      wrapper.setNamespace(namespace);
+      wrapper.setNamespace(namespace == null ? "" : namespace);
```

Comment:
If this is handled in the builder, does it need to happen here as well?

Reply:
If the DataboundSetter is not called then it could still be null.